docs/preview/features/secret-store/azure-functions.md

```diff
@@ -28,8 +28,7 @@ namespace MyHttpAzureFunction
     {
         public override void Configure(IFunctionsHostBuilder builder)
         {
-            IConfiguration config = builder.GetContext().Configuration;
-            builder.ConfigureSecretStore(stores =>
+            builder.ConfigureSecretStore((FunctionsHostBuilderContext context, IConfiguration config, SecretStoreBuilder stores) =>
```

Comment:
Technically , this is a breaking change...

We must also make sure that this change i reflected in our docs / code samples.

Reply:
Ah, I think not, bc we have only added an overload. So you could still use the one from before, it's just that there's now a new one that is 'better' so we could update the docs.

Reply:
Shall we update the documentation in this PR as well ?

Reply:
The documentation is already updated, this comment is even on a updated preview docs. 😉

Reply:
I need to pay more attention :D